models/DataObject/Classificationstore/KeyConfig/Dao.php

```diff
@@ -110,6 +110,12 @@ public function update()
             if (in_array($key, $this->getValidTableColumns(self::TABLE_NAME_KEYS))) {
                 if (is_bool($value)) {
                     $value = (int) $value;
+                    
+                    if(!$value) {
```

Comment:
However, I think we should check against
$key === "enabled" here

Reply:
> However, I think we should check against
> $key === "enabled"
> IN ADDITION here

I rebased it to 6.9. Also I added that check You asked.